server/rest/submission.py

```diff
@@ -128,6 +131,22 @@ def postSubmission(self, phase, folder, params):
             self.model('phase', 'challenge').requireAccess(
                 phase, user, level=AccessType.WRITE)
 
+        # Site admins may override the submission creation date
+        created = None
+        if 'date' in params:
+            if not user.get('admin'):
```

Comment:
I think we should use `self.requireAdmin()` here. This also makes me think we should have an optional message override to that method.